scripts/ci-e2e.sh

```diff
@@ -10,6 +10,22 @@ FORCE_REPO_UPDATE="${FORCE_REPO_UPDATE:-false}"
 
 export CAPM3RELEASEBRANCH="${CAPM3RELEASEBRANCH:-main}"
 
+# Extract release version from release-branch name
+if [[ "${CAPM3RELEASEBRANCH}" == release-* ]]; then
+  CAPM3_RELEASE_PREFIX="${CAPM3RELEASEBRANCH#release-}"
```

Comment:
Indentation is four spaces in this script.

Reply:
fixed